src/Giraffe/Giraffe.fsproj

```diff
@@ -48,6 +48,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.*" />
     <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="2.2.*" />
     <PackageReference Include="Microsoft.AspNetCore.ResponseCaching" Version="2.2.*" />
+    <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="1.2.2" />
```

Comment:
I hope the maintenance of this package gets some love, there's been no NuGet releases in over 2 years, but it looks like it could get better: https://github.com/microsoft/Microsoft.IO.RecyclableMemoryStream/issues/62

Reply:
If this came out of the Bing team I'd _hope_ that they're actively using it, meaning it's been stable.